go/protocol/keybase1/extras.go

```diff
@@ -691,7 +691,11 @@ func (s SigID) ToBytes() []byte {
 	if err != nil {
 		return nil
 	}
-	return b[0:SIG_ID_LEN]
+	n := len(b)
+	if n > SIG_ID_LEN {
```

Comment:
mixed opinions about this. while it's a little silly to panic here from a malformed sigID, i found a weird bug with sigID suffixes a few weeks ago because of this panic. i dunno - not a dealbreaker for me. just something to point out. there are pros to blowing up hard with something unexpected here.